cypress/e2e/LoginPage.cy.ts

```diff
@@ -75,7 +75,10 @@ describe('Login page', () => {
 
     cy.get('[data-cy="sign-in-button"]').click();
 
-    cy.get('[data-cy="account-button"]').click();
+    cy.url().should('not.include', '/login');
+
+    cy.get('[data-cy="account-button"]').should('have.text', 'Account').click();
```

Comment:
Mixing assertions with actions is not a good practice. The tests should follow the Arrange-Act-Assert pattern. What this does is Assert-Act-Assert.